upstream.wrap

```diff
@@ -0,0 +1,6 @@
+[wrap-file]
+subdirectory = imgui-sfml-2.1
```

Comment:
upstream.wrap has directory: ☒

Should this be `directory` instead?

Reply:
D'oh! :facepalm: Fixed.